src/pt/sssscanlator/src/eu/kanade/tachiyomi/extension/pt/sssscanlator/YomuComicsDto.kt

```diff
@@ -0,0 +1,190 @@
+package eu.kanade.tachiyomi.extension.pt.sssscanlator
+
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import keiyoushi.utils.tryParse
+import kotlinx.serialization.Serializable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+@Serializable
+data class HomeDto(
+    val featuredManga: List<MangaDto> = emptyList(),
+)
+
+@Serializable
+data class MangaDto(
+    val title: String,
+    val slug: String,
+    val cover: String? = null,
+) {
+    fun toSManga(): SManga = SManga.create().apply {
+        title = this@MangaDto.title
+        thumbnail_url = cover
+        url = "/obra/$slug"
+    }
+}
+
+@Serializable
+data class UpdatesDto(
+    val updates: List<UpdateMangaDto> = emptyList(),
+)
+
+@Serializable
+data class UpdateMangaDto(
+    val title: String,
+    val slug: String,
+    val cover: String? = null,
+) {
+    fun toSManga(): SManga = SManga.create().apply {
+        title = this@UpdateMangaDto.title
+        thumbnail_url = cover
+        url = "/obra/$slug"
+    }
+}
+
+@Serializable
+data class SearchResponseDto(
+    val results: List<SearchMangaDto> = emptyList(),
+    val pagination: PaginationDto? = null,
+)
+
+@Serializable
+data class SearchMangaDto(
+    val id: Int,
+    val name: String,
+    val slug: String? = null,
+    val coverImage: String? = null,
+) {
+    fun toSManga(): SManga = SManga.create().apply {
+        title = this@SearchMangaDto.name
+        thumbnail_url = coverImage
+        url = if (slug != null) "/obra/$slug" else "/series/$id"
+    }
+}
+
+@Serializable
+data class PaginationDto(
+    val page: Int,
+    val limit: Int,
+    val total: Int,
+    val pages: Int,
+)
+
+@Serializable
+data class DetailsResponseDto(
+    val obraData: ObraDataDto,
+)
+
+@Serializable
+data class ObraDataDto(
+    val id: Int,
+    val title: String,
+    val slug: String,
+    val synopsis: String? = null,
+    val cover: String? = null,
+    val status: String? = null,
+    val genres: List<String> = emptyList(),
+    val author: String? = null,
+    val artist: String? = null,
+    val chapters: List<ChapterDto> = emptyList(),
+) {
+    fun toSManga(): SManga = SManga.create().apply {
+        title = this@ObraDataDto.title
+        thumbnail_url = cover
+        description = synopsis
+        author = this@ObraDataDto.author
+        artist = this@ObraDataDto.artist
+        genre = genres.joinToString(", ")
```

Comment:
```suggestion
        genre = genres.joinToString()
```
`", "` is the default and can be omitted.